Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ControllerPoseSynchronizer.cs

```diff
@@ -214,5 +200,18 @@ public virtual void OnInputChanged(InputEventData<MixedRealityPose> eventData)
         }
 
         #endregion  IMixedRealityInputHandler Implementation
+
+        #region Obsolete
+
+        #pragma warning disable 0414
+        [SerializeField]
+        [HideInInspector]
+        [System.Obsolete("Use the Handedness property instead to get current handedness which is set by Controller attached")]
```

Comment:
I would seriously have to question why you are putting an obsolete flag on a PRIVATE field.  
Obsolete is for telling consumers of a function or property outside a class is it going to be deprecated.  Private fields do not fall in this category because they are private.  No need to make them obsolete.

Reply:
It was originally removed. Please read PR thread 
https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7012#discussion_r365003271